OmgvaPOS/src/ReservationManagement/Models/Reservation.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace OmgvaPOS.ReservationManagement.Models
 {
+    // TODO: add item to link what service will be provided with this Reservation
+    // TODO: add business or reuse business from item to protect this resource
```

Comment:
I am in favour of resuing business from item or employee fulfilling the reservation.